starter-code/css/main.css

```diff
@@ -6,3 +6,161 @@ Green: #00B172
 White: #FFF
 
 */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  color: 1a1a1a;
+}
+
+body {
+  width: 100%;
+  padding: 10px;
+  text-align: center;
+}
+
+header {
+  width: 100%;
+  height: 100px;
+  position: fixed;
```

Comment:
I would have gotten rid of the
        position : fixed
because it makes the header appear when you scroll down and would have added 
        background-color: #00B172;
        color: white;